src/components/Explore/TokenDetail.tsx

```diff
@@ -195,7 +196,7 @@ export default function LoadedTokenDetail({ address }: { address: string }) {
             {tokenName} <TokenSymbol>{tokenSymbol}</TokenSymbol>
           </TokenNameCell>
           <TokenActions>
-            <Share size={18} />
+            {tokenName && tokenSymbol && <ShareButton tokenName={tokenName} tokenSymbol={tokenSymbol} />}
```

Comment:
if the token name and token symbol don't exist the function should be bailed out of earlier on. what's the error state look like?

Reply:
<img width="351" alt="Screen Shot 2022-07-19 at 4 26 45 PM" src="https://user-images.githubusercontent.com/62825936/179842079-a9cb9066-b008-4684-88f4-eeaa9fc2f86e.png">